adapters/celestia/src/celestia.rs

```diff
@@ -370,14 +370,6 @@ impl<'a> TryFrom<&'a [u8]> for H160 {
     }
 }
 
-impl From<[u8; 32]> for H160 {
```

Comment:
I think `H160` might be is never used as an address (now that we have the correct Celestia address type). Can we remove it entirely?

Reply:
Sure! Done!